.github/workflows/lint.yaml

```diff
@@ -0,0 +1,23 @@
+name: lint
```

Comment:
nit: align name with job name `lint` vs. `sanity`

Only because the other workflows have aligned names.

Reply:
Done updated the filename as well.